src/config/routes.ts

```diff
@@ -8,21 +19,23 @@ export default function (app, passport, config, strategy, urlencodedParser) {
     })
   );
 
-  app.get(config.app.logoutPath, function (req, res) {
+  app.get(config.app.logoutPath, function (req: Request, res: Response) {
+    // TODO check this for logout
+    // @ts-ignore
     req.logout();
```

Comment:
This should be a passport function. After checking, it doesn't seem that passport has direct TS support either, but there is https://www.npmjs.com/package/@types/passport.

Reply:
Whoops! Just saw that you already included this in the package.json 👍